src/app/containers/NielsenAnalytics/Amp/index.test.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import AmpNielsenAnalytics from '.';
+
+describe('Assertions', () => {
+  it('should render Neilsen amp-analytics component', () => {
+    render(
+      <AmpNielsenAnalytics apid="apid" assetid="assetid" section="section" />,
+    );
+
+    const ampNeilsenEl = document.querySelector('amp-analytics');
```

Comment:
mind if we update these query selectors to use a RTL query instead? I think probably this one https://testing-library.com/docs/queries/bytestid e.g.

```html
<div data-testid="custom-element" />
```

```jsx
import {render, screen} from '@testing-library/react'

render(<MyComponent />)
const element = screen.getByTestId('custom-element')
```

Reply:
Go for it! I just copied the pattern from another test.